sormas-api/src/main/java/de/symeda/sormas/api/sample/SampleIndexDto.java

```diff
@@ -29,6 +29,7 @@
 import de.symeda.sormas.api.event.EventParticipantReferenceDto;
 import de.symeda.sormas.api.facility.FacilityHelper;
 import de.symeda.sormas.api.facility.FacilityReferenceDto;
+import de.symeda.sormas.api.labmessage.LabMessageReferenceDto;
```

Comment:
unused. Please always go throught the diff view yourself before requesting a review.